src/Uno.UI/UI/Xaml/ResourceDictionary.cs

```diff
@@ -188,7 +188,15 @@ private bool ContainsKeyMerged(object key)
 			return false;
 		}
 
-		private ResourceDictionary GetThemeDictionary() => GetThemeDictionary(Themes.Active) ?? GetThemeDictionary(Themes.Default);
+		private ResourceDictionary GetThemeDictionary(ElementTheme elementTheme = ElementTheme.Default)
+		{
+			var theme = Themes.Active;
+			if (elementTheme != ElementTheme.Default)
+			{
+				theme = elementTheme.ToString("g");
```

Comment:
Not sure if calling enum.ToString() has potential perf issues - a switch block might be safer.

Reply:
Adjusted :-)